test/pycheck/explain_test.py

```diff
@@ -54,6 +55,32 @@ def test_explain(cur: Cursor):
     assert "Total Time:" in plan
     assert "Output:" in plan
 
+    # Test for Json Output Format , psycopg internal convert json to dict
+    result = cur.sql("EXPLAIN (FORMAT JSON) SELECT count(*) FROM test_table")
+    assert len(result) == 1
+    assert type(result[0]) == dict
+    plan=json.dumps(result[0])
+    assert "UNGROUPED_AGGREGATE" in plan
+    assert "Total Time" not in plan
+    assert "Output" not in plan
+
+    result = cur.sql("EXPLAIN (VERBOSE, FORMAT JSON) SELECT count(*) FROM test_table")
+    assert len(result) == 1
+    assert type(result[0]) == dict
+    plan=json.dumps(result[0])
+    assert "UNGROUPED_AGGREGATE" in plan
+    assert "Total Time" not in plan
+    assert "Output" in plan
+
+    result = cur.sql("EXPLAIN (VERBOSE, ANALYZE, FORMAT JSON) SELECT count(*) FROM test_table")
+    assert len(result) == 1
+    assert type(result[0]) == dict
+    plan=json.dumps(result[0])
+    assert "EXPLAIN_ANALYZE" in plan
+    assert "UNGROUPED_AGGREGATE" in plan
+    assert "Total Time" in plan
+    assert "Output" in plan
```

Comment:
All these tests probably passed with the previous code too. Instead of converting the json to string it would be good to actually look into it to see that the keys are as expected.

Something like:
```python
    duckdb_plan = result[0]["Plan"]["DuckDB Execution Plan"]
    assert type(duckdb_plan) is list
    assert duckdb_plan[0]["name"] == "UNGROUPED_AGGREGATE"
```